broker-core/src/test/java/io/zeebe/broker/workflow/activity/ActivityTest.java

```diff
@@ -167,6 +168,35 @@ public void shouldUnsubscribeFromBoundaryEventTriggersOnTerminating() {
         WorkflowInstanceIntent.ELEMENT_TERMINATING, WorkflowInstanceIntent.ELEMENT_TERMINATED);
   }
 
+  @Test
+  public void shouldIgnoreTaskHeadersIfEmptyOrNull() {
```

Comment:
Can we split this into two tests, one testing empty, and the other testing null? Ideally we'd test also when the key and values are valid (i.e. key is empty or null, but value isn't, and vice versa).